src/test/mockFactory.ts

```diff
@@ -122,7 +122,7 @@ export const createTestServiceBuildAndUpdatePipelineYaml = (
                   ``,
                   `# Update HLD`,
                   `git checkout -b "$BRANCH_NAME"`,
-                  `../fab/fab set --subcomponent $SERVICE_NAME_LOWER image.tag=$(Build.SourceBranchName)-$(Build.BuildNumber)`,
+                  `../fab/fab set --subcomponent $(Build.Repository.Name).$SERVICE_NAME_LOWER.$(Build.SourceBranchName).chart image.tag=$(Build.SourceBranchName)-$(Build.BuildNumber)`,
```

Comment:
@evanlouie in the description you say 
> Calls to fab set in generated pipeline YAML files have been using the
incorrect component path to set -- only using the <service-name> instead of the
full path in the HLD of
<repository/application-name>-<service-name>-<ring-name>-chart. This fix
corrects the outputted fab set call.

However here we have `<application-name>-<service-name>-<ring-name> .chart` with a DOT chart. Is this correct?

Reply:
whoops. Wrote the comment incorrectly. We delimit the parts with `.` not `-`.
So `<application-name>.<service-name>.<ring-name> .chart`